tests/integration/transactions/test_clawback.py

```diff
@@ -71,3 +79,92 @@ async def test_basic_functionality(self, client):
             client,
         )
         self.assertTrue(response.is_successful())
+
+    # TODO: uncomment test when rippled Docker image supports MPT feature
+    # @test_async_and_sync(globals())
+    # async def test_mptoken(self, client):
+    #     wallet2 = Wallet.create()
```

Comment:
Will also need to pull in the changes from the verified credentials PR that edit the ci docker command for the entrypoint change